.github/labeler/review.yml

```diff
@@ -4,4 +4,6 @@
   - any-glob-to-any-file: "**/*"
 
 # Label to be removed when the PR is updated:
-"Reviewed: Action Needed": []
+"Reviewed: Action Needed":
+- changed-files:
+  - any-glob-to-any-file: "!**/*"
```

Comment:
Can you add a comment that this condition is always false because we want to always remove the label on updates?